<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinTestClass</class>
 <widget class="QWidget" name="WinTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>245</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>哈哈相机</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QCommandLinkButton" name="clb_selectImg">
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="text">
      <string>选择图片</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="description">
      <string>选择要处理的相片</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="3">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>魔幻引擎：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_engine">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>高级设置</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="sw_adv">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="pg_mesh">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QCommandLinkButton" name="clb_restore">
              <property name="font">
               <font>
                <family>Segoe UI</family>
               </font>
              </property>
              <property name="text">
               <string>复    原</string>
              </property>
              <property name="description">
               <string>恢复图像变形</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="pg_cover">
           <layout class="QFormLayout" name="formLayout">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>特效</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QComboBox" name="cmb_effect"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>相框</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="2">
             <widget class="QComboBox" name="cmb_cover"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QCommandLinkButton" name="clb_save">
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="text">
      <string>保   存</string>
     </property>
     <property name="description">
      <string>保存当前图片</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="winmagic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cb_engine</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>sw_adv</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
